components/widgets/menu/menu_items/menu_start_trial.tsx

```diff
@@ -30,16 +35,26 @@ const MenuStartTrial = (props: Props): JSX.Element | null => {
         dispatch(openModal({
             modalId: ModalIdentifiers.START_TRIAL_MODAL,
             dialogType: StartTrialModal,
+            dialogProps: {onClose: null},
+        }));
+    };
+
+    const openTrialBenefitsModal = () => {
+        dispatch(openModal({
+            modalId: ModalIdentifiers.TRIAL_BENEFITS_MODAL,
+            dialogType: TrialBenefitsModal,
         }));
     };
 
     const prevTrialLicense = useSelector((state: GlobalState) => state.entities.admin.prevTrialLicense);
-    const currentLicense = useSelector(getLicense);
+    const license = useSelector(getLicense);
     const isPrevLicensed = prevTrialLicense?.IsLicensed;
-    const isCurrentLicensed = currentLicense?.IsLicensed;
+    const isCurrentLicensed = license?.IsLicensed;
+
+    const isCurrentLicenseTrial = isTrialLicense(license);
 
     // Show this CTA if the instance is currently not licensed and has never had a trial license loaded before
-    const show = (isCurrentLicensed === 'false') && (isPrevLicensed === 'false');
+    const show = (isCurrentLicensed === 'false' && isPrevLicensed === 'false') || isCurrentLicenseTrial;
```

Comment:
Hello @pablovelezvidal , did we want to show this CTA if current license is a trial

Reply:
yes!

Reply:
yes, and it shows a different modal content. Check line 119